src/main/java/appeng/client/render/crafting/MolecularAssemblerRenderer.java

```diff
@@ -139,22 +143,30 @@ private void renderStatus(MolecularAssemblerTileEntity molecularAssembler, Matri
     }
 
     /**
-     * See above for when this can be removed. It creates a RenderType that is equivalent to
-     * {@link RenderType#getTranslucent()}, but enables alpha testing. This prevents the fully transparents parts of the
-     * rendered block model from occluding our particles.
+     * This class gives us access to the protected RenderState.TRANSLUCENT_TRANSPARENCY field.
      */
-    private static RenderType createRenderType() {
-        RenderState.TransparencyState TRANSLUCENT_TRANSPARENCY = ObfuscationReflectionHelper
-                .getPrivateValue(RenderState.class, null, "field_228515_g_");
-        RenderState.TextureState mipmapBlockAtlasTexture = new RenderState.TextureState(
-                AtlasTexture.LOCATION_BLOCKS_TEXTURE, false, true);
-        RenderState.LightmapState disableLightmap = new RenderState.LightmapState(false);
-        RenderType.State glState = RenderType.State.getBuilder().texture(mipmapBlockAtlasTexture)
-                .transparency(TRANSLUCENT_TRANSPARENCY).alpha(new RenderState.AlphaState(0.05F))
-                .lightmap(disableLightmap).build(true);
-
-        return RenderType.makeType("ae2_translucent_alphatest", DefaultVertexFormats.POSITION_COLOR_TEX_LIGHTMAP,
-                GL11.GL_QUADS, 256, glState);
+    private static class RenderTypeAccess extends RenderType {
```

Comment:
`MolecularAssemblerRenderType` or something similar is probably more fitting name.

Reply:
It's just used to fake the protected access, don't start bikeshedding names ^^